leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -139,20 +143,21 @@ var ElementInspector = Backbone.View.extend({
                     '<label style="float:left; font-size:0.8em;" id="repeat-begin3" class="repeat-select3">Absolute Length:</label>',
                     '<input style="float:right;"class="repeat-select3" id="repeat-end3" type="number" value="0">',
                     '<p id="noteRepeat" style="text-align:left; float:left; color:red; font-size:0.7em;">Note: Leave Absolute Length as 0 for unknown length. If Absolute Length is less than 0 or Repeat Count is less than 2, they will be set to 0 or 2 respectively.</p>',
-            // change to blue or change color to green
+                // change to blue or change color to green
                     '<button style="margin-top:10px;" id="constraint-add" class="inspector-btn small-btn green-btn">Add</button>',
                     '<button id="constraint-repeat" class="inspector-btn small-btn blue-btn">Set Repeats</button>',
                     '<button id="constraint-restart" class="inspector-btn small-btn red-btn">Clear</button>',
                 '</div>',
             '</div>',
             '<br>',
             '<canvas id="chart" width="240" height="240"></canvas>',
-
-    ].join(''),
+            
+            '</script>'].join(''),
 
     events: {
         'change #init-sat-value':'initSatValueChanged',
-
+        'click .intention-elements': 'switchModel',
```

Comment:
There is no intention elements class, and I don't think you should need to switch the model from inside the view